### ISSUE TITLE
Loading custom model

### ISSUE THREAD

**Issue by sulaimanshahbiz** [ultralytics/hub#355]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and found no similar feature requests.

### Description

Feature to load a model already trained externally. 

### Use case

The feature to include a way to upload our custom model, based on a custom dataset. This is because heavy training was already conducted before and we wish to use the existing platform with camera.
- so basically its the same as what the ultralytics hub app does, except we load our custom model into it.

**Comment by kalenmike** [comment]
@sulaimanshahbiz This is an internal challenge for us. If we enable model uploads from earlier version of the code we are not able to verify that they are indeed ultralytics models. This could potentially introduce bugs and issues that need to be reviewed which are not related to our code base.

We are planning to release the option to train HUB models with local dataset in the next two weeks. This will allow you to train models and preview them in HUB without needing to upload your datasets.

**Comment by lonnylundsten** [comment]
You can export your .pt model from ultralytics to .coreml format and, assuming you're using iOS, you can load your model into the Vision Detector app on your iPhone/pad/MacOS and use your device camera for testing realtime detection. https://apps.apple.com/us/app/vision-detector/id6443729650

**Comment by UltralyticsAssistant** [comment]
@lonnylundsten absolutely! Exporting your trained model into .coreml format is essentially how you can use it with iOS devices. After exporting your model, you can indeed load it into any application that accepts .coreml models, such as the Vision Detector app you mentioned. This will allow you to test real-time detection using your device's camera. It is a great way to deploy and test your model in real-world scenarios. Please follow the policies and guidelines set forth by each application for importing and utilizing custom machine learning models.
